geos/entrypoints.go

```diff
@@ -337,3 +347,29 @@ func MakeValid(g geom.Geometry, opts ...geom.ConstructorOption) (geom.Geometry,
 	})
 	return result, wrap(err, "executing GEOSMakeValid_r")
 }
+
+// The CoverageUnion function is used to union polygonal inputs that form a
+// coverage, which are typically provided as a GeometryCollection. This method
+// is much faster than other unioning methods, but there are some constraints
+// that must be met by the inputs to form a valid polygonal coverage. These
+// constraints are:
+//
+//  1. all input geometries must be polygonal,
+//  2. the inputs must not overlap, and
+//  3. the input boundaries must be noded in the same way.
```

Comment:
Does this sentence mean that the input boundaries share a common edge? Or is there a different meaning?